src/backend/sev/builder.rs

```diff
@@ -29,6 +29,64 @@ impl kvm::Hook for Sev {
         measure::Kind::Sha256
     }
 
+    fn shim_loaded(
+        &mut self,
+        _vm: &mut VmFd,
+        addr_space: &mut [u8],
+        shim: &Component,
+    ) -> Result<()> {
+        // The initial page tables of the shim must be corrected
+        // with the current CPU's C-bit indicating encrypted memory.
```

Comment:
In the case where the tenant has provided the shim over the network, how does the tenant code calculate the resulting measurement?

Reply:
Well, good question. So start with 16-bit mode?

Reply:
Maybe. Let's solve the problem later.

Reply:
We can exchange information about the c-bit in the client<->server communication and the client could self measure then with the same c-bit setup. This function is called before the `measurement`, so if we make c-bit a variable, this should work.